src/routes/assayclassifier/assay_types.yaml

```diff
@@ -416,7 +416,7 @@ seqFish:
 seqFish_pyramid:
     description: seqFISH [Image Pyramid]
     alt-names: []
-    primary: true
+    primary: false
```

Comment:
@jswelling according to @jpuerto-psc 's comment from the previous PR: https://github.com/hubmapconsortium/ingest-api/pull/441#discussion_r1425427802

Should we keep it as `primary: true` to be the same as the value specified in the deprecated yaml https://github.com/hubmapconsortium/search-api/blob/main/src/search-schema/data/definitions/enums/assay_types.yaml#L419?

Reply:
No, that value was wrong @yuanzhou - this should be set to false.